packages/valory/skills/transaction_settlement_abci/rounds.py

```diff
@@ -244,24 +241,25 @@ def end_block(
         if self.keeper_payload is None:  # pragma: no cover
             return self.synchronized_data, Event.FINALIZATION_FAILED
 
-        verification_status = VerificationStatus(self.keeper_payload["status_value"])
+        keeper_payload = self.keeper_payload["tx_data"]
+        status_value = keeper_payload["status_value"]
+        tx_hashes_history = keeper_payload["tx_hashes_history"]
+        serialized_keepers = keeper_payload["serialized_keepers"]
+        blacklisted_keepers = keeper_payload["blacklisted_keepers"]
+        received_hash = keeper_payload["received_hash"]
+
+        verification_status = VerificationStatus(status_value)
```

Comment:
keeper_payload is still just the content. I refer to https://github.com/valory-xyz/open-autonomy/pull/1727#discussion_r1065841695 where I suggest casting back to a "consensus" payload instance such that the structure and types of the data contained in the payload is explicitly provided.